File: java/Circle.java
/*
Problem:
Write a Java program to print the area and perimeter of a circle

*/
import java.util.Scanner;

public class Circle {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter the radius of the circle: ");
        double radius = scanner.nextDouble();

        double area = Math.PI * radius * radius;
        double perimeter = 2 * Math.PI * radius;

        System.out.println("Area of the circle: " + area);
        System.out.println("Perimeter of the circle: " + perimeter);
    }
}

File: python/Circle.py
"""
Problem:
Write a python program to print the area and perimeter of a circle

"""
import math

radius = float(input("Enter the radius of the circle: "))

area = math.pi * radius ** 2
print("Area of the circle is:", area)

perimeter = 2 * math.pi * radius
print("Perimeter (circumference) of the circle is:", perimeter)
